<launch>
    <arg name="mode" default="1" doc="mode:[UDP:1, SDK:2]"/>
    <arg name="show_debug_info" default="false" />
    <arg name="image_show_info" default="true" />
    <arg name="file_path" default="$(find hik_camera)"/>

  <node pkg="hik_camera" type="hikcam_node" name="hikcam" output="screen">
    <param name="mode" value="$(arg mode)" />
    <param name="show_debug_info" value="$(arg show_debug_info)" />
    <param name="image_show_info" value="$(arg image_show_info)" />
    <param name="file_path" value="$(arg file_path)" />


  </node>
</launch>
